assets/src/dashboard/components/infiniteScroller/index.js

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import {
+  useState,
+  useEffect,
+  useLayoutEffect,
+  useCallback,
+  useRef,
+} from 'react';
+import PropTypes from 'prop-types';
+import 'intersection-observer';
+
+const InfiniteScroller = ({
+  allDataLoadedMessage = 'No More Stories',
+  children,
+  handleGetData,
+  isAllDataLoaded,
+  loadingMessage = 'Loading...',
```

Comment:
Let's make sure these use the wp `___` helpers for text

Reply:
oh good catch!